news/20200427.misc

```diff
@@ -0,0 +1 @@
+Rename classmethods and internal attributes
```

Comment:
APIs names and attributes have been changed. This is a `major` change and so, the extension should reflect that.

Reply:
You're right. I'll change that.

Reply:
652b8b4d60235c8234c963cbe76c5de79e28790a